acme/acme/mixins.py

```diff
@@ -3,29 +3,7 @@
 from typing import Dict
 
 
-class VersionedLEACMEMixin:
```

Comment:
Another thing to warn about.

Reply:
I have created #9365 to track new things we want to deprecate in the next release, including `VersionedLEACMEMixin`.